osc_sdk/sdk.py

```diff
@@ -470,7 +470,9 @@ def format_data(self, parameters, key, value):
             self.format_data(parameters[head_key], queue_key, value)
         else:
             parameters[key] = (
-                value[1:-1].split(',') if value.startswith('[') else value
+                value[1:-1].split(',')
+                if not isinstance(value, bool) and value.startswith('[')
```

Comment:
la plus simple sera `if isinstance(value, str) and` ?

Reply:
mdr. Je n'y ai meme pas pensé une seconde... oui bien sur.

Reply:
Modifié.